packages/shopify-codemod/test/transforms/simplify-?=-if-else-statement.test.js

```diff
@@ -0,0 +1,8 @@
+import 'test-helper';
```

Comment:
Is there a really, really compelling reason to have `?=` in the file name?  I'd rather not.